Import trusted-types WPTs

lukewarlow opened the pull request:
#### d9824ab2c5a28936932ddbaa0fdb4c4593fecc60
<pre>
Import trusted-types WPTs
<a href="https://bugs.webkit.org/show_bug.cgi?id=293948">https://bugs.webkit.org/show_bug.cgi?id=293948</a>

Reviewed by Darin Adler.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/ea73257959cc22fcb20fe4bbceccfb3a2d5f6ee3">https://github.com/web-platform-tests/wpt/commit/ea73257959cc22fcb20fe4bbceccfb3a2d5f6ee3</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Element-setAttribute-respects-Elements-node-documents-globals-CSP-after-adoption-from-TT-realm-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-ServiceWorker.https.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicy-createXXX-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicy-createXXX.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-blocking.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-blocking.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-non-tt-policy-name-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-non-tt-policy-name.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Document-write.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-006.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-007-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-csp-headers.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/policy-without-return-value.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/policy-without-return-value.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001-outerHTML-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001-outerHTML.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002-outerHTML-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002-outerHTML.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-006-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-007-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-008.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-008.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-009.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-009.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-010-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-011-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-001.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-003.html.headers: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-003.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-005.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/csp-violations.js:
(trusted_type_violations_and_exception_for):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/event-handler-attributes.mjs:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/logMessage-module.sub.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigate-to-javascript-url-005-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigate-to-javascript-url.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-report-only-support.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-report-only-support.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.js:
(setWindowLocationToJavaScriptURLCode):
(const.kJavaScriptURLCode.executeJavaScript):
(createDefaultPolicy):
(async setLocationToJavaScriptURL.window.executeJavaScript):
(async setLocationToJavaScriptURL.window.executeModifiedJavaScript):
(async setLocationToJavaScriptURL.async return):
(async setLocationToJavaScriptURL):
(navigateToJavascriptURL.getAndPreparareNavigationElement):
(navigateToJavascriptURL): Deleted.
(getAndPreparareNavigationElement): Deleted.
(location.search.includes): Deleted.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/passthroughpolicy.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/script-messages.js: Added.
(create_html_script_with_trusted_source_text):
(create_html_script_with_untrusted_source_text):
(create_svg_script_with_trusted_source_text):
(create_svg_script_with_untrusted_source_text):
(script_messages_for):
(async script_message_for):
(async no_script_message_for):
(async base64_hash_for_inline_script):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-location.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-location.js.headers: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-multiple-violations.js: Added.
(promise_test.async let):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-multiple-violations.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-location.js: Added.
(passPlainStringToTrustedTypeSink):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-allow-duplicates-tt-policy-name.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-allow-duplicates-tt-policy-name.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-location.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-location.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-multiple-violations.js: Added.
(promise_test.async let):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-multiple-violations.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-none.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-none.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-allow-duplicates.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-allow-duplicates.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-none.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-none.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-wildcard.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-wildcard.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-location.js: Added.
(createForbiddenPolicy):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-create-policy-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-create-policy-worker.js.sub.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report.html.sub.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-create-policy.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-sink-mismatch-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-sink-mismatch-worker.js.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-sink-mismatch.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DedicatedWorker-constructor-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DedicatedWorker-constructor-worker.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DedicatedWorker-constructor.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-ServiceWorkerContainer-register-worker.https.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-ServiceWorkerContainer-register-worker.https.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-ServiceWorkerContainer-register.https.js: Added.
(getGlobalThisStr.includes):
(promise_test.async t):
(promise_test.async t.async assert_equals):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-SharedWorker-constructor.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-eval-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-eval-worker.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-eval.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-function-constructor-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-function-constructor-worker.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-function-constructor.js: Added.
(const.AsyncFunction):
(async const.GeneratorFunction):
(const.AsyncGeneratorFunction):
(async const):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-importScripts.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-importScripts.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-sandbox-no-allow-scripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-sandbox-no-allow-scripts.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_01-05-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_06-10-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_11-15-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_16-20-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_21-25-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_26-30-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_31-35-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_36-last-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-create-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-create-policy.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-sink-mismatch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-sink-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-create-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-create-policy.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-create-policy.html.sub.headers: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report.html.sub.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-sink-mismatch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-sink-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-sink-mismatch.html.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-DedicatedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-eval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-function-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-importScripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-children-change-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-children-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-children-change-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-children-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-eval.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-function-constructor.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-importScripts.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-setTimeout-setInterval.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-DedicatedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-eval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-function-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-importScripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-DedicatedWorker-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-DedicatedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-ServiceWorkerContainer-register.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-SharedWorker-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-SharedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-eval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-eval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-function-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-allow-scripts-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-allow-scripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-allow-scripts.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-no-allow-scripts-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-no-allow-scripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/w3c-import.log:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/tests-options.json:

* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/policy-without-return-value.sub-expected.txt: Added.

* LayoutTests/platform/win/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/300592@main">https://commits.webkit.org/300592@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/53c6bf6c1bc9b02975bc2c53db456fc601496960

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/122860 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/42572 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/33262 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/129494 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/74966 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/43291 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/51165 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/93387 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61990 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/125812 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/34514 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/109974 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/74013 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/33491 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/28131 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/72988 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/104214 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/28344 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/132233 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/49805 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/37917 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/101898 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/50182 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/106188 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/101767 "Found 2 new API test failures: WebKitGTK/TestInspector:/webkit/WebKitWebInspector/manual-attach-detach, TestWTF:CompletionHandlerDeathTest.MainThreadHandlerOnThreadAssertsDeathTest (failure)") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25925 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/47129 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/25315 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/46577 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/49663 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/49130 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/52481 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/50812 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/53c6bf6c1bc9b02975bc2c53db456fc601496960. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/46249

lukewarlow committed:
Import trusted-types WPTs
https://bugs.webkit.org/show_bug.cgi?id=293948

Reviewed by Darin Adler.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/ea73257959cc22fcb20fe4bbceccfb3a2d5f6ee3

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/Element-setAttribute-respects-Elements-node-documents-globals-CSP-after-adoption-from-TT-realm-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/ServiceWorkerContainer-register-from-ServiceWorker.https.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicy-createXXX-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicy-createXXX.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-blocking.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-blocking.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-non-tt-policy-name-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/TrustedTypePolicyFactory-createPolicy-non-tt-policy-name.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-Document-write.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-006.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-007-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/navigate-to-javascript-url-csp-headers.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/policy-without-return-value.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/policy-without-return-value.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001-outerHTML-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001-outerHTML.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002-outerHTML-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002-outerHTML.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-006-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-007-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-008.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-008.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-009.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-009.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-010-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-011-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/script-enforcement-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-001.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-003.html.headers: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-001.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-002.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-003.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-trusted-type-policy-creation-be-blocked-by-csp-005.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/csp-violations.js:
(trusted_type_violations_and_exception_for):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/event-handler-attributes.mjs:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/logMessage-module.sub.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigate-to-javascript-url-005-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigate-to-javascript-url.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-report-only-support.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-report-only-support.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.js:
(setWindowLocationToJavaScriptURLCode):
(const.kJavaScriptURLCode.executeJavaScript):
(createDefaultPolicy):
(async setLocationToJavaScriptURL.window.executeJavaScript):
(async setLocationToJavaScriptURL.window.executeModifiedJavaScript):
(async setLocationToJavaScriptURL.async return):
(async setLocationToJavaScriptURL):
(navigateToJavascriptURL.getAndPreparareNavigationElement):
(navigateToJavascriptURL): Deleted.
(getAndPreparareNavigationElement): Deleted.
(location.search.includes): Deleted.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/passthroughpolicy.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/script-messages.js: Added.
(create_html_script_with_trusted_source_text):
(create_html_script_with_untrusted_source_text):
(create_svg_script_with_trusted_source_text):
(create_svg_script_with_untrusted_source_text):
(script_messages_for):
(async script_message_for):
(async no_script_message_for):
(async base64_hash_for_inline_script):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-location.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-location.js.headers: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-multiple-violations.js: Added.
(promise_test.async let):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-002-worker-multiple-violations.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-sink-type-mismatch-violation-be-blocked-by-csp-location.js: Added.
(passPlainStringToTrustedTypeSink):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-allow-duplicates-tt-policy-name.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-allow-duplicates-tt-policy-name.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-location.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-location.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-multiple-violations.js: Added.
(promise_test.async let):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-multiple-violations.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-none.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-none.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-allow-duplicates.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-allow-duplicates.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-none.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name-none.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-tt-policy-name.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-wildcard.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-004-worker-wildcard.js.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/should-trusted-type-policy-creation-be-blocked-by-csp-location.js: Added.
(createForbiddenPolicy):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-create-policy-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-create-policy-worker.js.sub.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report.html.sub.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-create-policy.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-sink-mismatch-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-sink-mismatch-worker.js.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-check-report-sink-mismatch.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DedicatedWorker-constructor-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DedicatedWorker-constructor-worker.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-DedicatedWorker-constructor.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-ServiceWorkerContainer-register-worker.https.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-ServiceWorkerContainer-register-worker.https.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-ServiceWorkerContainer-register.https.js: Added.
(getGlobalThisStr.includes):
(promise_test.async t):
(promise_test.async t.async assert_equals):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-SharedWorker-constructor.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-eval-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-eval-worker.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-eval.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-function-constructor-worker.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-function-constructor-worker.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-function-constructor.js: Added.
(const.AsyncFunction):
(async const.GeneratorFunction):
(const.AsyncGeneratorFunction):
(async const):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-importScripts.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-reporting-for-importScripts.js.headers: Copied from LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-sandbox-no-allow-scripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/trusted-types-sandbox-no-allow-scripts.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-no-unsafe-eval.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-eval-reporting.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_01-05-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_06-10-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_11-15-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_16-20-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_21-25-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_26-30-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_31-35-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-navigation_36-last-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-report-only.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-create-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-create-policy.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-sink-mismatch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-DedicatedWorker-sink-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-create-policy-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-create-policy.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report.html.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-create-policy.html.sub.headers: Renamed from LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report.html.sub.headers.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-sink-mismatch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-sink-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-Window-sink-mismatch.html.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-check-report-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-clipping-of-sample.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-DedicatedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-eval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-function-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-importScripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-DedicatedWorker-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Element-outerHTML.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-children-change-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-HTMLScriptElement-children-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-children-change-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SVGScriptElement-children-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-eval.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-function-constructor.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-importScripts.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-ServiceWorker-setTimeout-setInterval.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-DedicatedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-eval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-function-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-importScripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-SharedWorker-setTimeout-setInterval-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-DedicatedWorker-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-DedicatedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-ServiceWorkerContainer-register.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-ServiceWorkerContainer-register.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-SharedWorker-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-SharedWorker-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-eval-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-eval.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-function-constructor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-for-Window-function-constructor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-allow-scripts-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-allow-scripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-allow-scripts.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-no-allow-scripts-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-sandbox-no-allow-scripts.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/w3c-import.log:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/tests-options.json:

* LayoutTests/platform/glib/imported/w3c/web-platform-tests/trusted-types/policy-without-return-value.sub-expected.txt: Added.

* LayoutTests/platform/win/TestExpectations:

Canonical link: https://commits.webkit.org/300592@main
